home.html

```diff
@@ -0,0 +1,155 @@
+                <li class="desktop-link">
+                    <a class="nav-link" href="#program">Program</a></li>
+                <li class="desktop-link">
+                    <a class="nav-link" href="#sponsor">Sponsor</a></li>
+                <li class="desktop-link">
+                    <a class="nav-link" href="#">Events</a></li>
+                <li class="desktop-link">
+                   <a class="nav-link" href="#">Join</a></li>
+                <li class="active-link">
+                    <a class="nav-link" href="#">Drummers Work Shop</a>
+                </li>
+            </ul>
+        </nav>
+        <nav class="mobile-menu">
+            <div class="mobile-menu-bar" id="humburger-icon">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </nav>
+    </header>
+</body>
+<main>
+    <section class="intro-section" id="home">
+        <h2 class="intro-head">"Drummers Making it Big!"</h2>
+        <h1 class="intro-heading">MUSIC FEST
+            <br>
+            DRUMMERS
+            <br>
+            SUMMIT 2022
+        </h1>
+        <p class="intro-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+            Impedit aspernatur deleniti assumenda dignissimos nulla fuga,
+            deserunt magnam? Tenetur vitae nisi optio veritatis numquam neque velit quasi.
+            Doloribus sed ratione nobis.</p>
+            <h3 id="date">2022.10.26(WED) ~ 28(FRI)</h3>
+            <p id="date-location">@ICC Central Business District, Abuja</p>
+    </section>
+
+    <section class="program-info" id="program">
+        <h2 id="main-program">Main Program</h1>
+        <div class="program-bar"></div>
+        <div class="program">
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_01.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Drum Chops Presentation</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/music-media.jpg" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Advertising & Media</h4>
+            <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                Molestiae qui odit fugiat nemo maiores earum, sunt saepe veniam sequi eum.</p>
+        </div>
+        <div class="program-list">
+            <div class="program-icon-wrapper">
+                <img class="program-icon" src="img/program_icon_03.png" alt="drum sticks icon">
+            </div>
+            <h4 class="program-topic">Building Connection</h4>
```

Comment:
- [ ] The cards in the section are articles and as such should be wrapped in a `article` tag